module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -2168,6 +2183,14 @@ protected function getStatusCodeItemNotForLoanOrLost($code, $data, $item)
      */
     protected function getStatusCodeItemTransfer($code, $data, $item)
     {
+        if (isset($data['to_library'])) {
+            $status = 'HoldingStatus::transit_to';
+            if (isset($data['datesent'])) {
+                $status = 'HoldingStatus::transit_to_date';
+            }
+            return $status;
```

Comment:
This is fine as-is, but if you wanted to reduce the code size, you could change this whole block to:

```
if (isset($data['to_library'])) {
    return isset($data['datesent']) ? 'HoldingStatus::transit_to_date' : 'HoldingStatus::transit_to';
}
```

Reply:
A separate question is whether the "on hold" status needs to be combined with the "transit to" status in some way. I see that there's logic related to this below.  Not sure how all the permutations work or what might take priority over what else. Feel free to ignore the question if you don't think this is important -- just asking because I noticed.